addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1432,13 +1433,19 @@ export class OdooEditor extends EventTarget {
     }
 
     multiselectionRefresh() {
-        this._collabSelectionsContainer.innerHTML = '';
+        // Refresh the selection but keep the relevant images to avoid flickering.
+        const avatars = [...this._collabSelectionInfos.values()].map(info => info.avatarElement);
+        for (const element of this._collabSelectionsContainer.querySelectorAll('*')) {
```

Comment:
I want to empty the whole thing, except for a few specific elements. It used to be `innerHTML = ''`. So yeah I do. It's not a big element though.

Reply:
I don't know, how would you remove all descendants but the ones that are in a given array without iterating and then filtering?
What I could probably do though is use `childNodes` instead of `querySelectorAll` actually, that would be a lot better. Would that suffice to you?

Reply:
Oh ok I missed the part where you checked `avatars.includes` so I wanted to use `oe-collaboration-caret-avatar` in the selector. Your suggestion is good, let's go for that.